handlers/download.go

```diff
@@ -85,7 +85,7 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error, logData
 		status = err.Code()
 	}
 	logData["setting_response_status"] = status
-	log.ErrorR(req, err, logData)
+	log.Event(req.Context(), "error setStatusCode", logData, log.Error(err))
```

Comment:
Probably want a more descriptive error message then <error function name>?

Reply:
actually, I think I should remove the log.error() part, because it is not an error-level log; we are actually setting a status code from an error.